editoast/src/fixtures.rs

```diff
@@ -78,6 +80,170 @@ pub mod tests {
         }
     }
 
+    #[fixture]
+    pub async fn train_schedule(
+        db_pool: Data<Pool<ConnectionManager<diesel::PgConnection>>>,
+        #[future] pathfinding: TestFixture<Pathfinding>,
+        #[future] timetable: TestFixture<Timetable>,
+        #[future] fast_rolling_stock: TestFixture<RollingStockModel>,
+    ) -> TestFixture<TrainSchedule> {
+        let pathfinding = pathfinding.await;
+        let timetable = timetable.await;
+        let rolling_stock = fast_rolling_stock.await;
+        let train_schedule = make_train_schedule(
+            db_pool.clone(),
+            pathfinding.id(),
+            timetable.id(),
+            rolling_stock.id(),
+        )
+        .await;
+        TestFixture {
+            model: train_schedule,
+            db_pool,
+            infra: None,
+        }
+    }
+
+    async fn make_train_schedule(
+        db_pool: Data<Pool<ConnectionManager<diesel::PgConnection>>>,
+        path_id: i64,
+        timetable_id: i64,
+        rolling_stock_id: i64,
+    ) -> TrainSchedule {
+        let train_schedule = TrainSchedule {
+            path_id,
+            timetable_id,
+            rolling_stock_id,
+            ..serde_json::from_str::<TrainSchedule>(include_str!("./tests/train_schedule.json"))
+                .expect("Unable to parse")
+        };
+        train_schedule.create(db_pool.clone()).await.unwrap()
+    }
+
+    #[derive(Debug)]
+    pub struct TrainScheduleProjectStudyScenarioTimetableInfraPathRollingStockFixture {
```

Comment:
Come on that's 70 characters :grin:
How about ?
```suggestion
    pub struct TrainScheduleFixtures {
```

Reply:
Fair enough :D